src/shared/modules/discovery/discoveryDuck.ts

```diff
@@ -164,111 +158,57 @@ export const discoveryOnStartupEpic = (some$: any, store: any) => {
         action.discoveryURL = discoveryURL
       }
 
-      return action
-    })
-    .merge(some$.ofType(USER_CLEAR))
-    .mergeMap(async (action: any) => {
-      // we can get data about which host different mechanisms, they are ranked in
-      // the following prioritization order
-      // 1. Url param - dbms
-      // 2. Url param - connectURL
-      // 3. database in discovery endpoint
-      // 4. Url param - discoveryURL
-
-      let dataFromForceUrl: DiscoverableData = {}
-
-      if (action.forceURL) {
-        const { username, protocol, host } = getUrlInfo(action.forceURL)
-
-        const discovered = {
-          username,
-          requestedUseDb: action.requestedUseDb,
-          host: `${protocol ? `${protocol}//` : ''}${host}`,
-          supportsMultiDb: !!action.requestedUseDb,
-          encrypted: action.encrypted,
```

Comment:
I'm a bit fuzzy on while all these discovered properties are needed, and whether they're actually used anywhere, but I guess it's fine for now.

Reply:
Yeah this turned into a bit of a rabbit hole trying to understand where they're all used. I'm somewhat certain that the encrypted key is not used here and that the restApi is used for the :http command. But didn't have enough time to dig into it fully